daemon/stop.go

```diff
@@ -38,9 +38,10 @@ func (daemon *Daemon) containerStop(container *container.Container, seconds int)
 		return nil
 	}
 
+	stopSignal := container.StopSignal()
 	// 1. Send a SIGTERM
-	if err := daemon.killPossiblyDeadProcess(container, container.StopSignal()); err != nil {
-		logrus.Infof("Failed to send SIGTERM to the process, force killing")
+	if err := daemon.killPossiblyDeadProcess(container, stopSignal); err != nil {
+		logrus.Infof("Failed to send signal %d to the process, force killing", stopSignal)
```

Comment:
we could translate it to a `named` signal, but I prefer the number,  `translation` will add some extra complication, it doesn't worth. And some other place just use the number https://github.com/docker/docker/blob/master/daemon/kill.go#L42
WDYT?